lib/dest/index.js

```diff
@@ -12,11 +12,21 @@ function dest(outFolder, opt) {
   }
 
   function saveFile(file, enc, cb) {
+    /* jshint validthis: true */
+    var stream = this;
+
+    function complete(err, file) {
+      cb(err, file);
+      if (!err) {
```

Comment:
seems very similar to my proposition in #120 but can you enlighten me how the error propagates or not in this situation?

is dealing with the stream being `done` or rather flushed as I proposed not ideal ?